BREAKING.md

```diff
@@ -46,6 +47,11 @@ This field has been deprecated and will be removed in a future breaking change.
 `enableShareLinkWithCreate` feature gate has been deprecated and will be removed in a future breaking change. If you wish to enable creation of a sharing link along with the creation of Fluid file, you will need to provide `createShareLinkType:ISharingLinkKind` input to the `createOdspCreateContainerRequest` function and enable the feature using `enableSingleRequestForShareLinkWithCreate` in `HostStoragePolicy`
 ### Add assertion that prevents sending op while processing another op
 `preventConcurrentOpSend` has been added and enabled by default. This will run an assertion that closes the container if attempting to send an op while processing another op. This is meant to prevent non-deterministic outcomes due to concurrent op processing.
+
+### Add fluidInvalidSchema errorType to DriverErrorType enum
+Added fluidInvalidSchema errorType in DriverErrorType enum. This error happens when non-fluid file
+was mistook as a Fluid file, and is unable to be opened. The innerMostErrorCode will also be "fluidInvalidSchema". For consumers, this needs to be handled as a separate error type, or have a separate case in order to not throw the error.
```

Comment:
I'd add here that it's not breaking change yet, but if clients do not add handling of this error, their existing version of applications may start receiving this error in the future and not handle it correctly